ui_framework/components/local_nav/_local_breadcrumbs.scss

```diff
@@ -25,34 +24,26 @@
         }
       }
     }
-
-    &:last-child {
-      .kuiLocalBreadcrumb__link {
-        color: $localNavTextColor;
-        cursor: default;
-
-        &:hover {
-          text-decoration: none;
-        }
-
-        @include darkTheme {
-          color: $localNavTextColor--darkTheme;
-        }
-      }
-    }
   }
 
     .kuiLocalBreadcrumb__link {
+      $breadcrumbLinkColor: #328CAA;
```

Comment:
Sounds good, will change. BTW the CSS file is all just generated stuff so we can ignore it during code reviews. :)